tests/test_joint.py

```diff
@@ -309,7 +317,7 @@ def test_fixedjoint():
     dampingforce = nu * normal_relative_vel * distance / end_distance
     contactforce = elasticforce - dampingforce
 
-    fxjt = FixedJoint(k, nu, kt)
+    fxjt = FixedJoint(k, nu, kt, nut, rest_rotation_matrix=np.eye(3))
```

Comment:
Can we add more test cases with the non-initially-aligned scenario?

Reply:
I agree with @skim0119 . New `fixed joint` class is capable of more than connecting two aligned systems. Maybe add one more test for connecting two rods that are perpendicular to each other.

Reply:
Thanks. I addressed this in https://github.com/GazzolaLab/PyElastica/pull/135/commits/d84f5f1234aad142f732c15908ba7e40dd2eb466